Content.Server/_DV/Abilities/Kitsune/KitsuneFoxSystem.cs

```diff
@@ -0,0 +1,32 @@
+using Content.Server.Polymorph.Components;
+using Content.Server.Polymorph.Systems;
+using Content.Shared._DV.Abilities.Kitsune;
+using Content.Shared.Actions;
+using Content.Shared.Damage.Systems;
+using Content.Shared.Polymorph;
+using Content.Shared.Stunnable;
```

Comment:
Remove using directives only used by unused `_actions`
```suggestion
using Content.Server.Polymorph.Components;
using Content.Server.Polymorph.Systems;
using Content.Shared._DV.Abilities.Kitsune;
using Content.Shared.Damage.Systems;
using Content.Shared.Stunnable;
```